src/recruiter/recruiter.service.ts

```diff
@@ -44,17 +48,100 @@ export class RecruiterService {
     const oneRecruiter = await this.dataSource
       .getRepository(Recruiter)
       .createQueryBuilder('recruiter')
-      .where('recruiter.id = :id', {id})
-      .andWhere('recruiter.registerToken = :registerToken', {registerToken})
+      .where('recruiter.id = :id', { id })
+      .andWhere('recruiter.registerToken = :registerToken', { registerToken })
       .getOne();
 
-      return oneRecruiter ? {
-        isOk: true,
-        id: oneRecruiter.id,
-        role: 'Recruiter',
+    return oneRecruiter
+      ? {
+          isOk: true,
+          id: oneRecruiter.id,
+          role: 'Recruiter',
+        }
+      : {
+          isOk: false,
+        };
+  }
+
+  async changeStatus(id: string, status: string) {
+    //@TODO wysyłanie informacji do admina, ewentualna zmiana nazewnictwa w case'ach, lepsza walidacja błędów
+    //@TODO upewnienie się, że rekruter po wpisaniu w ściężkę id innego studenta niż ma w dodanych dostanie informację o braku dostępu
+    const foundStudentImport = await StudentImport.findOne({
```

Comment:
nie wiem czy to dobrze, aby szukać po id StudentImport
trzeba na to spojrzeć od strony FE, bo chyba łatwiej będzie przeszukać po id Student
w każdym razie trzeba coś przyjąć i się tego trzymać

Reply:
Nie za bardzo rozumiem o co chodzi. W sensie, frontend nie dostanie od nas obu ID, tylko jedno, więc które by to nie było nie powinno im byc trudniej bądź łatwiej. Uważam, że studentImport powstaje pierwsze i to id z tej tabeli powinno być takie główne i odsyłane.

Chyba że czegoś nie rozumiem :/

A poza tym uważam, że fajnie jakby id w obu tych tabelach było takie same, ale nie wiem jak to przerobić i czy w ogóle warto na tym etapie się w to bawić.

Reply:
W sumie powinno być po angielsku :) no ale...
My na BE operujemy na 2-óch tabelach (obiektach/encjach), które dotyczą kursanta. A na FE dostaną z BE jeden obiekt. No i pytanie, które id (Student, czy StudentImport) ma być tym głównym? 

Wydaje mi się, że idzie zrobić tak, aby Student i StudentImport miały to samo id. Wystarczy w encji Student zamienić dekorator PrimaryGeneratedColumn na PrimaryColumn przy id. Potem przy tworzeniu Student trzeba pamiętać, aby przypisać id z StudentImport do id Student. 
Tylko nie wiem, czy tak powinno się robić.